webhooks/webhooks.constant.ts

```diff
@@ -0,0 +1,91 @@
+export class WebhooksConstant {
+    static readonly webhooksUrl = '/f/page?ixPlugin=21&pg=pgPluginConfig';
+
+    static readonly attributes = Object.freeze({
+        text: Object.freeze({
+            header: 'Webhooks',
+            addNewHook: 'Add New Hook',
+            editHook: 'Editing Hook',
+            deleteHook: 'Delete Hook',
+            ok: 'o',
+            viewLog: 'View Log',
+            import: 'Import',
+            export: 'Export',
+            xmlImport: 'XML Import',
+            importOk: 'OK',
+            importSuccessfulMessage: 'Import Successful: Webhooks.xml',
+            returnToConfigurationPage: 'Return to Configuration Page'
+        }),
+        class: Object.freeze({
+            headerClass: 'headline',
+            dialogTitle: 'dialog-title'
+        }),
+        id: Object.freeze({
+            editable: 'editable',
+            allEventsCb: 'idAllFogBugzEvents',
+            caseEventsCb: 'idCaseEvents',
+            discussTopicEventsCb: 'idDiscussTopicEvents',
+            sourceControlCommitCb: 'idSourceControlCommit',
+            wikiEventsCb: 'idWikiEvents',
+            timeIntervalEventsCb: 'idTimeIntervalEvents',
+            milestoneEventsCb: 'idMilestoneEvents',
+            webhookTable: 'staticURLTrigger',
+            hookTypeDropdownArrow: 'idDropList_idSelectTemp_0_oButton',
+            hookTypeDropdown: 'idDropList_idSelectTemp_0_oDropList',
+            mainArea: 'mainArea',
+            logTable: 'staticLog',
+            timeIntervalAdded: 'idTimeIntervalAdded',
+            timeIntervalEdited: 'idTimeIntervalEdited',
+            timeIntervalDeleted: 'idTimeIntervalDeleted'
+        }),
+        name: Object.freeze({
+            urlTextbox: 'sURL',
+            filterTextbox: 'sFilter',
+            nameTextbox: 'sName',
+            okButton: 'sOK'
+        }),
+        title: Object.freeze({
+            edit: 'Edit',
+            delete: 'Delete'
+        }),
+        type: Object.freeze({
+            file: 'file'
+        })
+    });
+
+    static readonly testData = Object.freeze({
+        url: 'http://www.aurea.com',
+        url2: 'https://webhook.site/',
+        filter: ' PriorityID="1"',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/davidgunadi/hello-world/pullRequest?prid=8400130)
```suggestion
        filter: " PriorityID=\"1\"",
```